obspy/signal/tests/test_cross_correlation.py

```diff
@@ -6,22 +6,222 @@
                         unicode_literals)
 from future.builtins import *  # NOQA
 
+import ctypes as C
+import numpy as np
 import os
 import unittest
+import warnings
 
 from obspy import UTCDateTime, read
+from obspy.core.util.libnames import _load_cdll
 from obspy.core.util.testing import ImageComparison
-from obspy.signal.cross_correlation import xcorr_pick_correction
+from obspy.signal.cross_correlation import (xcorr, xcorr_pick_correction,
+                                            xcorr_3c, xcorr_max,
+                                            _xcorr_padzeros, _xcorr_slice)
 
 
 class CrossCorrelationTestCase(unittest.TestCase):
+
     """
     Cross corrrelation test case
     """
+
     def setUp(self):
         # directory where the test files are located
         self.path = os.path.join(os.path.dirname(__file__), 'data')
         self.path_images = os.path.join(os.path.dirname(__file__), 'images')
+        self.a = np.sin(np.linspace(0, 10, 101))
+        self.b = 5 * np.roll(self.a, 5)
+        self.c = 5 * np.roll(self.a[:81], 5)
+
+    def test_xcorr(self):
+        """
```

Comment:
Moving around these tests makes it hard to review.. are there changes in the old tests?